### ISSUE TITLE
Sampling rate

### ISSUE THREAD

**Issue by KamijoYo** [matsim-org/matsim-code-examples#281]
I have the impression that some of the studies using MATSim are simulating at a sample rate of 10%. Are there any studies that have verified that 10% is enough?

**Comment by Janekdererste** [comment]
**TL;DR** No, it depends.

As far as I know there is no structured research on what is a good sample size.

Usually it depends on what you want to simulate, since depending on the sample size one agent represents multiple persons (e.g in a 1% sample one agent represents 100 persons). That said, conducting a simulation which investigates car, bike or pedestrian traffic is probably fine with a sample size of 1%.

If you want to simulate public transport or taxi traffic a larger sample size is probably necessary. If you'd run a 1% sample with taxi traffic, one agent entering a taxi would mean 100 persons enter this vehicle. At this point the abstraction doesn't make sense anymore.

**Comment by KamijoYo** [comment]
Thank you very much for your reply.

There are cases where only a very small sample rate is used in simulations using taxis, and it is unlikely that appropriate results have been obtained.

**Comment by kainagel** [comment]
For _single passenger_ taxi, http://dx.doi.org/10.14279/depositonce-8270  Section 8.4.3 says something about scenario scaling.  The result is that 10% leads to results not much different from 100%.  We are also running 1% and the results look "structurally" ok.  I will make a mental note that we should also do quantitative comparisons.

I think that one can also argue what happens.  Assume we start with 100%, single passenger, and taxis parked on a 1km x 1km grid.  After a request, the taxi first will have an empty drive of in the average a bit more than 0.5km.  For 10%, the grid would be 3.16km x 3.16 km, for 1% it would be 10km x 10km, i.e. initial drives would be a bit more than 1.6km and a bit more than 5km.  This means pickup times should in the average be around 1min/4min/12min.  I would expect that in simulation reality, the differences between the sample sizes will be even smaller, since vehicles are not homogeneously distributed.

The argument completely breaks down with pooling, i.e. multiple passengers in the same vehicle.  Here, the proba to find a matching trip should go down with the same scaling as the sample size, i.e. a 1% sample should under-estimate pooling probability by a factor of 100.  I think.

**Comment by jfbischoff** [comment]
For most cases, we are running a 10 % model and find the results useful in aggregate (e.g, not for every single train load, but overall per line or station). We also use a 25% model for which allows a better spot on, e.g., small train stations.  Every now and then there is a call for a 100% model, but the added value would IMO not justify the huge computational load.

---
As to taxis: In my experience, fleet and operational characters scale really well as long as the density of requests is high, i.e., you are looking at cities.
For areas with fewer population density, YMMV. But then again, less densely populated areas often allow a higher sampling size :-)

**Comment by KamijoYo** [comment]
Thank you very much for introducing the paper. 
That is the reference I seek.

I plan to handle pooling in areas with low population density, so I'm thinking about simulating with 100% travel demand.

**Comment by jfbischoff** [comment]
>I plan to handle pooling in areas with low population density, so I'm thinking about simulating with 100% travel demand.

That seems to be a good idea, yes.
